cranelift/Cargo.toml

```diff
@@ -4,7 +4,7 @@ authors = ["The Cranelift Project Developers"]
 version = "0.59.0"
 description = "Binaries for testing the Cranelift libraries"
 license = "Apache-2.0 WITH LLVM-exception"
-documentation = "https://cranelift.readthedocs.io/"
+documentation = "https://github.com/bytecodealliance/wasmtime/blob/master/cranelift/docs/index.md"
```

Comment:
Is this right? I get a 404.

Reply:
Never mind... I see what's happening down below.

Reply:
It'll *become* right when this PR lands :sunglasses: